### DISCUSSION TITLE
Under the hood, the markdown gets converted into rst for Sphinx?

### DISCUSSION THREAD

**Discussion by zhen-insitro** [sphinx-doc/sphinx#12931] [Q&A] [answered]
rst file is the default format used by Sphinx, and Sphinx also [supports markdown using a thrid-party pacakge](https://www.sphinx-doc.org/en/master/usage/markdown.html). Just out of curiosity, does markdown file is converted to rst for Sphinx under the hood?

**Comment by AA-Turner** [comment, marked as answer]
Regardless of which markup format is used, the text is parsed into a tree of nodes that is then used internally.

A

**Comment by zhen-insitro** [comment]
Thanks @AA-Turner ! For the process of parsing the text into a tree of nodes, which text format is more robust for Sphinx? 

For some cases, if I wrote the complex math formula in Markdown, the formula can't be rendered properly on the webpage. But the markdown file display the formula correctly.  So I was wondering the above question.
